src/features/expense/ui/UnReviewedExpenseView/UnReviewedExpenseView.test.tsx

```
@@ -0,0 +1,61 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render } from '@testing-library/react';
+import { describe, it, vi } from 'vitest';
+
+import * as expenseHooks from '@/features/expense/api/useExpenseQuery';
+import { QueryClientTestProvider } from '@/test/utils/createQueryClient';
```

ESLint

[error] 'QueryClientTestProvider' is defined but never used.

(@typescript-eslint/no-unused-vars)